client/extensions/woocommerce/app/dashboard/index.js

```diff
@@ -132,6 +133,8 @@ class Dashboard extends Component {
 			setupChoicesLoading,
 		} = this.props;
 
+		const adminURL = get( selectedSite, 'options.admin_url', '' );
```

Comment:
Is there any chance we will indeed get the default empty string value here? If so should we ensure we have a string with length prior to attempting to window.location = ''?

Reply:
I don't think so, but we could test for that and console an error perhaps?

Reply:
Added `warn` in https://github.com/Automattic/wp-calypso/pull/21297/commits/79f4d91320a4f456fe0d4f7a2400fb40395ff473